internal/operators/ocs/manifest.go

```diff
@@ -204,91 +200,92 @@ spec:
-              matchExpressions:
-              - key: app
-                operator: In
-                values:
-                - rook-ceph-osd
-            topologyKey: kubernetes.io/hostname
-          weight: 100
-    preparePlacement:
-      nodeAffinity:
-        requiredDuringSchedulingIgnoredDuringExecution:
-          nodeSelectorTerms:
-          - matchExpressions:
-            - key: node-role.kubernetes.io/worker
-              operator: Exists
-      podAntiAffinity:
-        preferredDuringSchedulingIgnoredDuringExecution:
-        - podAffinityTerm:
-            labelSelector:
-              matchExpressions:
-              - key: app
-                operator: In
-                values:
-                - rook-ceph-osd-prepare
-            topologyKey: kubernetes.io/hostname
-          weight: 100
-
-    portable: false
-
-    replica: 3
-`
+//
+//const ocsSc = `apiVersion: ocs.openshift.io/v1
+//
+//kind: StorageCluster
+//
+//metadata:
+//
+//  name: ocs-storagecluster
+//
+//  namespace: openshift-storage
+//
+//spec:
+//
+//  labelSelector:
+//
+//    matchExpressions:
+//      - key: node-role.kubernetes.io/worker
+//        operator: In
+//        values:
+//        - ""
+//
+//  manageNodes: false
+//  monDataDirHostPath: /var/lib/rook
+//
+//  storageDeviceSets:
+//
+//  - count: {{.OCSDisks}}
+//
+//    dataPVCTemplate:
+//
+//      spec:
+//
+//        accessModes:
+//
+//        - ReadWriteOnce
+//
+//        resources:
+//
+//          requests:
+//
+//            storage: "1"
+//
+//        storageClassName: 'localblock-sc'
+//
+//        volumeMode: Block
+//
+//    name: ocs-deviceset
+//
+//    placement:
+//      nodeAffinity:
+//        requiredDuringSchedulingIgnoredDuringExecution:
+//          nodeSelectorTerms:
+//          - matchExpressions:
+//            - key: node-role.kubernetes.io/worker
+//              operator: Exists
+//      podAntiAffinity:
+//        preferredDuringSchedulingIgnoredDuringExecution:
+//        - podAffinityTerm:
+//            labelSelector:
+//              matchExpressions:
+//              - key: app
+//                operator: In
+//                values:
+//                - rook-ceph-osd
+//            topologyKey: kubernetes.io/hostname
+//          weight: 100
+//    preparePlacement:
+//      nodeAffinity:
+//        requiredDuringSchedulingIgnoredDuringExecution:
+//          nodeSelectorTerms:
+//          - matchExpressions:
+//            - key: node-role.kubernetes.io/worker
+//              operator: Exists
+//      podAntiAffinity:
+//        preferredDuringSchedulingIgnoredDuringExecution:
+//        - podAffinityTerm:
+//            labelSelector:
+//              matchExpressions:
+//              - key: app
+//                operator: In
+//                values:
+//                - rook-ceph-osd-prepare
+//            topologyKey: kubernetes.io/hostname
+//          weight: 100
+//
+//    portable: false
+//
+//    replica: 3
+//`
```

Comment:
Is this PR still a work in progress?
If so, can you add `[WIP]` to the title, if not can you remove this commented out block?

Reply:
yes, I have added the label